pkg/uefi/meregion_test.go

```diff
@@ -91,3 +92,26 @@ func TestFindFPTSignature(t *testing.T) {
 		})
 	}
 }
+
+func TestParseFPT(t *testing.T) {
+	t.Run("parse ME FPT", func(t *testing.T) {
+		fixtureFile := "../../data/PurleySiliconBinPkg/MeFirmware/IgnitionFirmware/MeRegion.bin"
+		meRegion, err := os.ReadFile(fixtureFile)
+		if err != nil {
+			t.Fatalf("could not read test fixture %q", fixtureFile)
+		}
+		expectedEntries := 10
+
+		fpt, err := NewMEFPT(meRegion)
+		if err != nil {
+			t.Errorf("expected no error reading a valid ME FPT, got %v", err)
```

Comment:
This should be log.Fatalf, you're not going anywhere when this fails.

Reply:
That is what this question was about:
https://github.com/linuxboot/fiano/pull/445#discussion_r1794210468